src/rust/engine/grpc_util/src/tls.rs

```diff
@@ -33,71 +49,105 @@ impl TryFrom<Config> for ClientConfig {
     let tls_config = ClientConfig::builder().with_safe_defaults();
 
     // Add the root certificate store.
-    let mut tls_config = match config.certificate_check {
+    let tls_config = match config.certificate_check {
       CertificateCheck::DangerouslyDisabled => {
         let tls_config = tls_config.with_custom_certificate_verifier(Arc::new(NoVerifier));
-        if let Some(MtlsConfig { key, cert_chain }) = config.mtls {
+        if let Some(MtlsConfig { cert_chain, key }) = config.mtls {
```

Comment:
Why change the order? It would be nice to have one consistent field order everywhere. If this makes more sense, why not change the order in the definition of MtlsConfig?

Reply:
Switched order.